src/diffusers/models/unet_2d_condition.py

```diff
@@ -145,6 +145,7 @@ def __init__(
         time_cond_proj_dim: Optional[int] = None,
         conv_in_kernel: int = 3,
         conv_out_kernel: int = 3,
+        transformer_num_layers: int = 1,
```

Comment:
```suggestion
        transformer_num_layers: Optional[int] = None,
```

Not all unets have a transformer so IMO can be a bit confusing to see this set to 1 by default